plugins/record-hook/rollup.config.mjs

```diff
@@ -23,15 +23,12 @@ const external = [
 
 function commonPlugins(browser, umd = false) {
   return [
-    !umd
-      ? peerDepsExternal({
-          includeDependencies: true,
-        })
-      : undefined,
+    !umd ? peerDepsExternal() : undefined,
```

Comment:
`includeDependencies: true,` is deprecated and using it results in `const sheet = api.sheets.get(...)` to return `TypeError: Cannot read properties of undefined (reading 'get')`